src/constants.ts

```diff
@@ -0,0 +1,6 @@
+export class Actions {
```

Comment:
I saw your comment about not being able to use an enum here because TypeScript does not support string enums. I think you might be misunderstanding the point of enums, though: the whole idea is to *not* use strings for comparison, but instead to use integers. A drop-in replacement for what you have here would be:
```TypeScript
export enum Actions {
  rent,
  return,
  add,
  edit
}
```
I think this is functionally the same as what you created here with a `class`, but I see it as a slightly cleaner implementation.